src/overrides.json

```diff
@@ -0,0 +1,8 @@
+{
+  "decimal_types": {
+    "BigDecimal": {
+      "p": 60,
+      "d": 2
```

Comment:
As we are following USDC with 6 decimals maybe we should do that here as well, or we accept losing precision if someone sends 0.00005 for example. Depends on what we want 😆

Reply:
good point, this was done because I initially thought we would only care about 2 decimal places but you're right I think it makes more sense to go with 6. I'm even wondering if we should stick to 18 since if we want to better future proof